app/src/main/res/values/strings.xml

```diff
@@ -343,17 +345,14 @@
     <string name="FFMI_formula">FFMI [kg/m2] = weight [kg] × (1 − (body fat [%] / 100)) / (height [m])²</string>
     <string name="no_fat_available">no fat available</string>
     <string name="indexLabel">Index</string>
-    <string name="what_type_of_exercise">What type of exercise ?</string>
+    <string name="what_type_of_exercise">Exercise type?</string>
     <string name="exercise_type">Exercise Type:</string>
     <string name="staticExercise">Isometric</string>
-    <string name="SecondsLabel_short">Secs</string>
-    <string name="imported_successfully"> imported successfully</string>
-    <string name="import_failed"> import failed</string>
-    <string name="import_new_exercise_first">(Make sure to import new exercises first)</string>
+    <string name="SecondsLabel_short">Sec.</string>
+    <string name="imported_successfully"> imported</string>
+    <string name="import_failed">could not import files</string>
+    <string name="import_new_exercise_first">(Ensure all new exercises are imported first)</string>
```

Comment:
Somewhat ambigous

Reply:
Yes, it would require a more detailled guide on how to use the import feature. Thing is that, if you import **records** first, it will automatically create **exercices**. So if then, you import the "real exercices", it will create conflicts.
That's the purpose of this warning.